### ISSUE TITLE
Error with 'LeftCtrl+Num+' key

### ISSUE THREAD

**Issue by vdomos** [krayon/ratslap#16]
Hi,

I have this error when assign a button to **LeftCtrl+Num+** (zoom+ in FIrefox), **LeftCtrl+Num-** is ok.

```
$ ./ratslap --modify F3 --G7 LeftCtrl+Num+ --G6 LeftCtrl+Num- --print F3 --select F3                  
RatSlap v0.2.2-tags/0.2.2^0 (BUILT: 2018-07-11 11:06:16+0200)
Copyright (C) 2016 Todd Harbour
Linux configuration tool for Logitech mice (currently only G300/G300S)
https://gitlab.com/krayon/ratslap/
Found Logitech G300s (046d:c246) @ 0x5568a0d53ce0
Detaching kernel driver...
Modifying Mode: F3
    Setting button 7: LeftCtrl+Num+
20180712T132246+0200 [E]           main.c:01101:set_mode_button ERROR: Invalid modifier (Num) specified: LeftCtrl+Num+
    Setting button 6: LeftCtrl+Num-
Saving Mode: F3
Printing Mode: F3
  Colour:              cyan
  Report Rate:          500
  DPI #1:               500
  DPI #2:        (DEF) 1000
  DPI #3:              1500
  DPI #4:              2500
  DPI Shift:           NOT SET
  Left Click (But1):   Button1
  Right Click (But2):  Button2
  Middle Click (But3): Button3
  G4:                  Button6
  G5:                  Button7
  G6:                  LeftCtrl + Num-
  G7:                  Button9
  G8:                  LeftCtrl + Tab
  G9:                  LeftCtrl + LeftShift + Tab
Mode Selection Specified: F3
Selecting Mode: F3
Attaching kernel driver...

```

**Comment by krayon** [comment]
TRACKED AS QB#125 : https://bugs.qdnx.org/issue/125 ( Error assigning Num+ )

**Comment by krayon** [comment]
Fixed in 2c237a28eb8d45088ab178a707930060124ac34d
